experimental/CHANGELOG.md

```diff
@@ -31,6 +31,24 @@ All notable changes to experimental packages in this project will be documented
 ### :house: (Internal)
 
 * refactor: Simplify the code for the `getEnv` function [#4799](https://github.com/open-telemetry/opentelemetry-js/pull/4799) @danstarns
+* refactor: remove "export *" in favor of explicit named exports [#4880](https://github.com/open-telemetry/opentelemetry-js/pull/4880)
+  * Packages updated:
+    * api-events
+    * api-logs
+    * exporter-logs-otlp-grpc
+    * exporter-trace-otlp-grpc
+    * exporter-trace-otlp-http
+    * opentelemetry-browser-detector
+    * opentelemetry-exporter-metrics-otlp-grpc
```

Comment:
We didn't end up doing the exporter packages after all, we just added an ignore on them right? Maybe we can skip those altogether here.

Reply:
yep, there's just a docs update, we can skip that in the entry :slightly_smiling_face:

Reply:
Removed!